src/sentry/workflow_engine/processors/workflow.py

```diff
@@ -312,5 +312,14 @@ def process_workflows(event_data: WorkflowEventData) -> set[Workflow]:
                 "detector_type": detector.type,
             },
         )
+    if features.has("organizations:workflow-engine-metric-alert-dual-processing-logs"):
+        # in order to check if workflow engine is firing 1:1 with the old system, we must only count once rather than each action
+        metrics.incr(
+            "workflow_engine.process_workflows.fired_actions",
+            tags={
+                "detector_type": detector.type,
+                "organization_id": detector.project.organization_id,
+            },
+        )
```

Comment:
thoughts on pulling out the `org_id` and setting the tag on every metric? something like:
```py
organization_id = None
if features.has("organizations:workflow-engine-metric-alert-dual-processing-logs"):
    organization_id = detector.project.organization_id
    
        metrics.incr(
            "workflow_engine.process_workflows.triggered_actions",
            amount=len(actions),
            tags={"detector_type": detector.type, "organization_id": organization_id},
        )
```

Reply:
I am making this a separate metric because I don't want to have the amount being set to the number of actions as that's not how we track it in the existing system.

Reply:
sorry for the confusion, the action one is just meant to illustrate an existing metric decorated with the organization id.

mostly just thinking that we could just add the organization id to all the existing metrics if we pull out the check for the feature and assign the org id to a separate variable.

Reply:
🤔 all the other metrics in this function pass an `amount` that's possibly larger than 1, and I only ever want to count a metric issue firing or resolving once otherwise the metrics might not line up